CHANGELOG.md

```diff
@@ -50,6 +50,11 @@ The format is based on [Keep a Changelog].
 
 ### Changed
 
+- `IBMQBackend.jobs()` now accepts a new boolean parameter `descending`,
```

Comment:
The changelog doesn't need cross references, since it won't format correctly in say github. I'll add the cross references when I do the release note.